CI/file_check.sh

```diff
@@ -0,0 +1,110 @@
+#!/bin/bash
+
+FILENAME=$1
+
+DEST="\$(DESTDIR)"
+
+while IFS= read -r LINE; do
+
+    case $LINE in
+
+        "DESTDIR="*)
+            IFS='='
+            read -a DESTDIR_VALUE <<< $LINE
+            ;;
+    
+        *"install -d"*)
+            IFS=' '
+            read -a COMMAND_ARGUMENTS <<< $LINE
+            LENGTH=${#COMMAND_ARGUMENTS[@]}
+
+            if [[ ! ${COMMAND_ARGUMENTS[$LENGTH-1]} == *"(DESTDIR)"* ]]; then
+
+                if [[ ! -d "${COMMAND_ARGUMENTS[$LENGTH-1]}" ]]; then
+                    echo "Directory ${COMMAND_ARGUMENTS[$LENGTH-1]} DOES NOT exists."
+                    exit 1
+                else
+                    echo "Directory ${COMMAND_ARGUMENTS[LENGTH-1]} exists."
+                fi
+
+            else
+
+                if [[ ! -d "${COMMAND_ARGUMENTS[$LENGTH-1]/${DEST}/${DESTDIR_VALUE[1]}}" ]]; then
+                    echo "Directory ${COMMAND_ARGUMENTS[$LENGTH-1]/${DEST}/${DESTDIR_VALUE[1]}} DOES NOT exists."
+                    exit 1
+                else
+                    echo "Directory ${COMMAND_ARGUMENTS[$LENGTH-1]/${DEST}/${DESTDIR_VALUE[1]}} exists."
+                fi
+
+            fi
+            ;;
+
+        *"install -D -m"*)
+            IFS=' '
+            read -a COMMAND_ARGUMENTS <<< $LINE
+            LENGTH=${#COMMAND_ARGUMENTS[@]}
+            FILE_PATH=${COMMAND_ARGUMENTS[$LENGTH-1]}
+
+            if [[ $FILE_PATH == *"/" ]]; then
+
+                IFS='/'
+                read -a COPIED_FILE <<< ${COMMAND_ARGUMENTS[$len-2]}
+                LENGTH=${#COPIED_FILE[@]}
+
+                if [[ ! -f "$FILE_PATH${COPIED_FILE[$LENGTH-1]}" ]]; then
+                    echo "File $FILE_PATH${COPIED_FILE[$LENGTH-1]} DOES NOT exists"
+                    exit 1
+                else
+                    echo "File $FILE_PATH${COPIED_FILE[$LENGTH-1]} exists"
+                fi
+
+            else
+                if [[ ! -f "$FILE_PATH" ]]; then
+                    echo "File $FILE_PATH DOES NOT exists"
+                    exit 1
+                else
+                    echo "File $FILE_PATH exists"
+                fi
+            fi
+            ;;
+
+        *"systemctl enable"*)
+
+            LENGTH=$(wc -w <<< $LINE)
+            NEWLINE=$(cut -d " " -f 3-$LENGTH <<< $LINE)
+            IFS=' '
+            read -a COMMAND_ARGUMENTS <<< $NEWLINE
+        
+            for ARGUMENT in "${COMMAND_ARGUMENTS[@]}"; do
+
+                if [[ $ARGUMENT == *".service" ]]; then
+
+                    RESULT=$(sudo systemctl status $ARGUMENT)
```

Comment:
Instead of using multiple 'IF' condition, you can concatenate 2 'grep' commands, so line 82 may be something like:
`if $(systemctl status ${ARGUMENT} | grep 'Loaded' | grep -q 'enabled') `, where `-q ` parameter of last grep is used to not output the matched text, but only the exit code. With this, grep returns zero if it found something vs. non-zero exit code if it hasn't found anything. In an if statement, a zero exit code is mapped to "true" and a non-zero exit code is mapped to false. So after this IF, you can put directly your output message - `echo "Service $ARGUMENT is enabled"`